Client/core/CMainMenu.cpp

```diff
@@ -221,6 +224,53 @@ CMainMenu::CMainMenu(CGUI* pManager)
     m_pLatestNews->SetProperty("InheritsAlpha", "False");
     m_pLatestNews->SetVisible(false);
 
+    static const std::array<std::pair<std::string, std::string>, 6> m_socialMediaLinks{{
```

Comment:
`string_view` or `const char*` pls no `std::string` needed here.
Also, it can be marked `constexpr` (Eg.: `static constexpr`)

Reply:
done